RESOURCES.md

```diff
@@ -567,3 +567,43 @@ This resource provides the following ensurable methods for managing storage volu
 * `get_connectable_volume_templates` - Gets the storage volume templates that are available on the specified networks based on the storage system port's expected network connectivity. If no storage volume templates exist that meet the specified connectivity criteria, an empty collection will be returned.
 
 Example file: [volume_template.pp](examples/volume_template.pp)
+
+# Resources for the Puppet Module for HPE Image Streamer
```

Comment:
Not sure about this naming. The module is, in the end, oneview-puppet, or the long version "Puppet Module for HPE OneView". 
I guess I'd prefer just a `# Resources for managing the HPE Image Streamer` or `# HPE Image Streamer resources`